.github/workflows/main.yml

```diff
@@ -37,15 +37,15 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: 3.8
-      # TODO: Find out about the AmazonRootCA1.pem file and sort below section accordingly
+      # TODO: Find out about the rds-ca-2019-root.pem file and sort below section accordingly
```

Comment:
this is now done isn't it?

Reply:
Yes this is complete. Was originally unsure of the certificates requirement when writing the workflow.